### ISSUE TITLE
Giveth one pager

### ISSUE THREAD

**Issue by Cotabe** [Giveth/GIVeconomy#876]
GM!

Potential partners have asked me if we can share a one-pager about Giveth. When they do not know what Giveth is, the pitch deck or other resources are just too much information.

So I designed a one-pager, that Rodri helped me make more beautiful.

I would love to get some feedback from fundraising @GriffGreen @spectoregia & comms @laurenluz @c0ric0ri

https://drive.google.com/file/d/1xRQNPQCF8BigK1__b2PLqAyVq-9GEfUa/view?usp=sharing

**Comment by laurenluz** [comment]
@Cotabe - I think @danelsuga can help clean this up a little, some gramatical errors I spotted, and some confusing parts:
- staking, not stacking
- Regen Economy description on Nonprofit side... value recirculates before exiting the community, etc. it's kind of unclear what this means... and it doesn't really describe what it is.

Small suggestion:
- maybe have "done" "in progress" and "to be developed" as 3 options... to indicated that GIVFi is IPR, whereas regen economies is somewhere in the future.

Also it's tough to leave feedback on this while looking at a PDF since there's not much ability to comment, maybe, if Suga can help, you could work out a call time w/ her and improve things live 🤔

**Comment by laurenluz** [comment]
@rodricastillo9013  can you share the access to the Figma here so we can edit the copies? (from @Cotabe )

**Comment by Cotabe** [comment]
Here are some suggestions for additional edits to the one-pager:

- [ ] Add the fully diluted market cap @Cotabe 
- [ ] Correct grammar & capitalization (The part of “the incentives for giving...” would not be written this way by a native speaker. @danelsuga @c0ric0ri (can you help me with this?)
- [ ] Add quadratic funding & Referral programs instead of having GIVfi twice. @Cotabe 
- [ ] Change funders for donors. 
- [ ] More feedback on content, style and wording. Maybe @divine-comedian can give more feedback.

**Comment by danelsuga** [comment]
In addition to Lauren's above corrections:

Philanthropy is **broken.** 
Funders have no upside: ...impact investing **grows** 10% annually. 
It's a lose-lose **situation**, **with no** incentives...
Update **giving.**

To add to "Better for nonprofits"
- no intermediaries between donation and project

Regen economies on "funders" side: "with **the chance** of"

Changing lives section:
not capitalized: Social, Land trusts, Skateboarding

"Impact numbers" chart: no periods.

I find this a really odd statement: We believe the best time to disrupt...

**Comment by danelsuga** [comment]
Feedback would be a lot easier if the document could be edited directly.
Thanks @Cotabe !

**Comment by Cotabe** [comment]
I left some comments on the Figma and asked @rodricastillo9013 to give edit access to @danelsuga.

I also made this doc with the texts that were raised on the feedback and were trickier to fix on the comments. Would love your thoughts on it @danelsuga 

https://docs.google.com/document/d/1G8HmYyNRBGdbRNzd1jwVxDg_PTMW_GVAekLg2R1hy80/edit?usp=sharing

**Comment by danelsuga** [comment]
I'm a little confused: the "one pager copies to improve" is separate from the edits you will make that @laurenluz and I included above? I have reviewed the "one pager copies to improve". Will try to come back to it again to support with regen economies explanation and catchy end CTA.

**Comment by Cotabe** [comment]
@danelsuga 

Yes, I already made comments on the Figma for the edits that were clear and easy to implement.

But some of them, like the explanation of Regen Economies or the last comment were easier to edit on a doc. Then I will aggregate the comments on the Figma. I replied to the comments.

If there is anything else please add it to the document, here or maybe make a loom video :)

**Comment by danelsuga** [comment]
Thanks, @Cotabe ! OK, great.
